### PR TITLE
Adds "AWS Instance Metadata Discovery" TTP

### PR DESCRIPTION
**Pull request by w0rk3r**

### CODE REVIEW

File: discovery/4d5de7b1-4c9f-4a46-ae0b-1622bd6931e0.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+id: 4d5de7b1-4c9f-4a46-ae0b-1622bd6931e0
+name: AWS Instance Metadata Discovery
```
Comment (khyberspache) [inline discovery/4d5de7b1-4c9f-4a46-ae0b-1622bd6931e0.yml:2]: `Retrieve AWS instance metadata`

File: discovery/4d5de7b1-4c9f-4a46-ae0b-1622bd6931e0.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+description: Retrieve Instance Metadata for AWS Instances
```
Comment (khyberspache) [inline discovery/4d5de7b1-4c9f-4a46-ae0b-1622bd6931e0.yml:3]: Use this (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) to expand on this description (see my comments on your other pull requests)

File: discovery/4d5de7b1-4c9f-4a46-ae0b-1622bd6931e0.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  tags: []
+tactic: discovery
+technique:
+  id: T1082
+  name: "System Information Discovery"
+platforms:
+  windows:
+    psh:
+      command: |
+        Invoke-RestMethod -Uri http://169.254.169.254/latest/dynamic/instance-identity/document
```
Comment (khyberspache) [inline discovery/4d5de7b1-4c9f-4a46-ae0b-1622bd6931e0.yml:17]: This doesn't seem to match the name of the TTP. There is a specific metadata endpoint `http://169.254.169.254/latest/meta-data/` - this TTP is pulling the identity document (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-identity-documents.html).

**Review by khyberspache — CHANGES_REQUESTED** [review]
The name on this TTP doesn't seem to match the TTP you're using. Can you clarify the name & description what you're trying to accomplish?

File: discovery/4d5de7b1-4c9f-4a46-ae0b-1622bd6931e0.yml (hunk 3 above)
Comment (w0rk3r) [reply discovery/4d5de7b1-4c9f-4a46-ae0b-1622bd6931e0.yml:17]: Yeah man, sorry...Thanks for pointing it out! Fixed

**Review by khyberspache — APPROVED** [review]

**Pull request merged**